plugins/experimental/uri_signing/config.c

```diff
@@ -298,7 +298,8 @@ read_config_from_json(json_t *const issuer_json)
     }
 
     json_t *jwk_obj;
-    cjose_err jwk_err = {0};
+    cjose_err jwk_err;
+    memset(&jwk_err, 0, sizeof(cjose_err));
```

Comment:
I don't think these changes are necessary (but harmless).  https://godbolt.org/z/xceT5f5z8

Reply:
True.  I wouldn't have changed these if clang wasn't complaining about those {0} initializers.  This was from centos llvm toolset 7.